magento/magento2
# Include email address in error message for customer AlreadyExistsException

### Description
When importing a large number of customers, the process can fail with an exception when there are duplicate customers being imported / which collide with existing users in a different store. Tracking down what entry in the import can be difficult. This pull request adds the offending email address to the exception message to aid in identifying the problem data.

### Related Pull Requests
*None*


### Fixed Issues
*None yet*, please create if required.

### Manual testing scenarios
1. Log into admin
1. Navigate to Customers -> All Customers
1. Click 'Add New Customer' button in the menu
1. Fill the form
1. Click 'Save Customer'
1. Notice success message: "You saved the customer."
1. Click 'Add New Customer' button in the menu
1. Fill the form with the same data as before
1. Click 'Save Customer'
1. Observe error message
  Before: `A customer with the same email address already exists in an associated website.`
  After: `A customer with the same email address (customer@example.com) already exists in an associated website.`

These testing instructions show just one place where this error message shows up. In this location the email address in question isn't far away from the error message and therefore it's not difficult to determine the problem data. However when performing a customer import, the error may be one of thousands of addresses, and therefore more difficult to determine without the information being added to the message in this pull request.

### Questions or comments
*None*

### Contribution checklist (*)
